src/dmd/link.d

```diff
@@ -1260,6 +1365,12 @@ version (Windows)
                 else if (!x64 && FileName.exists(FileName.buildPath(sdk, "kernel32.lib"))) // SDK 7.1 or earlier
                     return sdk;
             }
+
+            // try mingw fallback relative to phobos library folder that's part of LIB
+            Strings* libpaths = FileName.splitPath(getenv("LIB"));
+            if (auto p = FileName.searchPath(libpaths, r"mingw\kernel32.lib", false))
```

Comment:
This doesn't match the 2.079-beta1 installer that places all platform libraries alongside phobos64.lib. Was this a deliberate choice?